### ISSUE TITLE
can not reach 66%

### ISSUE THREAD

**Issue by CangHaiQingYue** [LevinJ/SSD_tensorflow_VOC#10]
Thanks for your great work. But I still have some issues needs your help:
I traian on voc07+12_train datasets for 160000 steps(train 30000+finetune 130000), and finally I found that mAP is 57% below. I dont konw why, Is my datasets(about 2.4GB) wrong? 
![default](https://user-images.githubusercontent.com/30284428/31869121-70aef1fc-b7d8-11e7-9f3b-19a2448bf489.png) This is evaluted on voc07_test(trained on voc07+12_train)
Another question is that I trianed on voc07_train dataset, and  got a higher mAP than trained on voc07+12_train.
![default](https://user-images.githubusercontent.com/30284428/31869253-55eae794-b7d9-11e7-92a6-0203ab592788.png)

**Comment by LevinJ** [comment]
Hi,  can you also post your loss chart here?  To be honest, it's a bit hard to say outright where the problem is. The codes should be all right, I remember there a few other people repeated the result with the same code base.

You second chart does not look right to me, in that the accuracy actually decrease from 25k to 40k iteration.. One possible cause is that you might forget to adjust learning rate when you start fine tuning all the parameters in the model.  Please also double check  the readme and ensure you followed the steps as suggested.

**Comment by CangHaiQingYue** [comment]
Thank you! I will check my work. After finish, I will report my new result.

**Comment by CangHaiQingYue** [comment]
Thank you! I will check my work. After finish, I will report my new result.@Levin Jian

**Comment by CangHaiQingYue** [comment]
@LevinJ  Thank you! I will check my work, and report new result.

**Comment by CangHaiQingYue** [comment]
I’ve got 63% mAP. Thanks a lot.
![default](https://user-images.githubusercontent.com/30284428/32046913-ffce7d08-ba76-11e7-8694-653fe71ec68e.png)
![default](https://user-images.githubusercontent.com/30284428/32047071-77dbde26-ba77-11e7-9ffd-7f2926ad24c6.png)
PS: The mAP we use reference AP_VOC2012/mAP   not  AP_VOC2007/mAP ?
@LevinJ

**Comment by Levin** [comment]
hey
i dont know who you are but i think that you typed in the wrong mail adress
short question : what are yu doing with these Maps ?
good luck
cheers

2017-10-26 12:01 GMT+02:00 CangHaiQingYue <notifications@github.com>:

**Comment by LevinJ** [comment]
Hi Levin over there, these charts are used to track the training process of a neural network. Have a nice day :)

**Comment by Levin** [comment]
see yaa and good luck



2017-12-01 2:40 GMT+01:00 Levin Jian <notifications@github.com>:
